cpp/tests/CMakeLists.txt

```diff
@@ -69,18 +88,18 @@ function(ConfigureTest CMAKE_TEST_NAME)
             GTest::gtest_main
             NCCL::NCCL
     )
-
-    add_test(NAME ${CMAKE_TEST_NAME} COMMAND ${CMAKE_TEST_NAME})
-
     set_target_properties(
         ${CMAKE_TEST_NAME}
             PROPERTIES INSTALL_RPATH "\$ORIGIN/../../../lib")
 
-    install(
-        TARGETS ${CMAKE_TEST_NAME}
-        COMPONENT testing
-        DESTINATION bin/gtests/libcugraph
-        EXCLUDE_FROM_ALL)
+    rapids_test_add(
+        NAME ${CMAKE_TEST_NAME}
+        COMMAND ${CMAKE_TEST_NAME}
+        GPUS ${_CUGRAPH_TEST_GPUS}
+        PERCENT ${_CUGRAPH_TEST_PERCENT}
+        INSTALL_COMPONENT_SET testing #bin/gtests/libcugraph
```

Comment:
```suggestion
        INSTALL_COMPONENT_SET testing
```

Reply:
Unless you intended to keep that to document where the file was going? That's not obvious to me, but if that's the intent feel free to leave it as is. That information is otherwise provided by the install_relocatable calls at the bottom.

Reply:
This was leftover as I was porting to `install_relocatable` will remove